src/stm.py

```diff
@@ -136,13 +123,26 @@ def _cbf_welch(volume, fps, max_freq = 20):
     N = nextpow2(volume.shape[0])
     f, Pxx = signal.welch(volume, fs = fps, window = "hann", nfft = N,
         return_onesided = True, axis = 0)
-    max_freq_indices = Pxx.argmax(axis = 0)
-    heatmap = f[max_freq_indices]
-    heatmap[heatmap > max_freq] = max_freq
-    return heatmap
+    return get_best_volumes(Pxx,f,max_freq,sequence)
 
 # Utilities
 
+def get_best_volumes(pxx, f, max_freq, top = 5):
```

Comment:
Several items:
 - rename function to `_get_best_volumes`, given its role as a utility that is only called by other utilities.
 - rename `top` to `n_maps`, again consistent with previous naming conventions
 - rename `pxx` and `f` to something more meaningful to the user (e.g., `pow_spec` and `freq_bins`, respectively).